icu4j/main/tests/core/src/com/ibm/icu/dev/test/message2/Mf2FeaturesTest.java

```diff
@@ -115,6 +117,7 @@ public void testDefaultLocale() {
                 .arguments(arguments)
                 .expected(expectedRo)
                 .build());
+        Locale.setDefault(Locale.US);
```

Comment:
Please restore the original default, rather than setting a hardcoded en-US locale.